docs/howto/configure/pangeo-requester-pays-access.md

```diff
@@ -0,0 +1,103 @@
+# Pangeo Data Access via Requester Pays
```

Comment:
Perhaps we should make this generic instead of Pangeo-specific, since I could imagine other hubs would want the same setup?